projects/wallet-template/src/containers/WalletPopup/pages/Accounts/AccountDetails.tsx

```diff
@@ -0,0 +1,67 @@
+import { Clipboard, CheckCircle, ArrowLeft } from "lucide-react"
+import React, { useState } from "react"
+import { useParams, useNavigate } from "react-router-dom"
+import { useCopyToClipboard } from "usehooks-ts"
+
+export const AccountDetails: React.FC = () => {
+  const navigate = useNavigate()
+  const { accountId } = useParams<{ accountId: string }>()
+  const [copied, setCopied] = useState(false)
+  const [_, copy] = useCopyToClipboard()
+
+  if (!accountId) {
+    return null
+  }
+
+  const onCopyToClipboard = async () => {
+    setCopied(true)
+    setTimeout(() => {
+      setCopied(false)
+    }, 2000)
+    await copy(accountId)
+  }
```

Comment:
the `setTimeout` needs to be removed when the component is unmounted.
If not, `setCopied` could be called in an unmounted component and there will be a React warning in the console